src/test/java/org/cactoos/bytes/HexOfTest.java

```diff
@@ -66,13 +68,19 @@ public void validHex() throws Exception {
         ).affirm();
     }
 
-    @Test(expected = IOException.class)
     public void invalidHexLength() throws Exception {
-        new HexOf(new TextOf("ABF")).asBytes();
+        new Assertion<>(
+            "Must invalid hex length",
+            () -> new HexOf(new TextOf("ABF")).asBytes(),
+            new Throws<>(IOException.class)
+        ).affirm();
     }
 
-    @Test(expected = IOException.class)
     public void invalidHex() throws Exception {
-        new HexOf(new TextOf("ABG!")).asBytes();
+        new Assertion<>(
+            "Must invalid hex",
+            () -> new HexOf(new TextOf("ABG!")).asBytes(),
+            new Throws<>(IOException.class)
```

Comment:
@baudoliver7 It would be better to also match message of exception

Reply:
@andreoss Done.